### PR TITLE
Update use-mail-protection-reports.md

### PR DESCRIPTION
**Pull request by andypunt**
Updating to use docs links

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/monitoring/use-mail-protection-reports.md [hunk 1]
```diff
@@ -28,18 +28,18 @@ If you were viewing mail protection reports in the Exchange admin center, they'v
...
+|**Advanced Threat Protection and email security reports** <br/> Email security and threat protection reports (including malware, spam, phishing, and spoofing reports).| In the Security & Compliance Center, go to **Reports** > **Dashboard**.|[View reports for Office 365 Advanced Threat Protection](https://docs.microsoft.com/en-us/office365/securitycompliance/view-reports-for-atp)  <br/>  [View email security reports in the Security & Compliance Center](https://docs.microsoft.com/en-us/office365/securitycompliance/view-email-security-reports)|
+|**Mail flow** <br/> Information about sent and received email messages, recent alerts, top senders and recipients, email forwarding reports, and more .| In the Security & Compliance Center, go to **Mail flow** > **Dashboard**.|[Mail flow insights in the Office 365 Security & Compliance Center](https://docs.microsoft.com/en-us/office365/securitycompliance/mail-flow-insights-v2)|
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/monitoring/use-mail-protection-reports.md]:
```suggestion
|**Mail flow** <br/> Information about sent and received email messages, recent alerts, top senders and recipients, email forwarding reports, and more.| In the Security & Compliance Center, go to **Mail flow** > **Dashboard**.|[Mail flow insights in the Office 365 Security & Compliance Center](https://docs.microsoft.com/en-us/office365/securitycompliance/mail-flow-insights-v2)|
```

Comment (andypunt) [reply Exchange/ExchangeOnline/monitoring/use-mail-protection-reports.md]: I think it looks good to me.

File: Exchange/ExchangeOnline/monitoring/use-mail-protection-reports.md [hunk 2]
```diff
@@ -28,18 +28,18 @@ If you were viewing mail protection reports in the Exchange admin center, they'v
 To get to the Security & Compliance Center, visit [https://protection.office.com](https://protection.office.com), and sign in using your work or school account.
 
 > [!NOTE]
-> You must be an Office 365 global administrator or have appropriate permissions assigned in order to use the Security & Compliance Center. For more details, see [Permissions in the Office 365 Security & Compliance Center](https://support.office.com/article/d10608af-7934-490a-818e-e68f17d0e9c1).
+> You must be an Office 365 global administrator or have appropriate permissions assigned in order to use the Security & Compliance Center. For more details, see [Permissions in the Office 365 Security & Compliance Center](https://docs.microsoft.com/en-us/office365/securitycompliance/permissions-in-the-security-and-compliance-center).
```
Comment (nenonix) [inline Exchange/ExchangeOnline/monitoring/use-mail-protection-reports.md]:
```suggestion
> You must be an Office 365 global administrator or have appropriate permissions assigned in order to use the Security & Compliance Center. For more details, see [Permissions in the Office 365 Security & Compliance Center](https://docs.microsoft.com/office365/securitycompliance/permissions-in-the-security-and-compliance-center).
```

File: Exchange/ExchangeOnline/monitoring/use-mail-protection-reports.md [hunk 3]
```diff
@@ -28,18 +28,18 @@ If you were viewing mail protection reports in the Exchange admin center, they'v
...
-|**Mail flow** <br/> Information about sent and received email messages, recent alerts, top senders and recipients, email forwarding reports, and more .| In the Security & Compliance Center, go to **Mail flow** > **Dashboard**.|[Mail flow insights in the Office 365 Security & Compliance Center](https://support.office.com/article/beb6acaa-6016-4d54-ba7e-3d6d035e2b46)|
+|**Threat management dashboard** (this is also referred to as the **Security dashboard** and the **Threat Intelligence dashboard**). <br/> Threat detections, malware trends, top targeted users, details about sent and received email messages, and more.|In the Security & Compliance Center, go to **Threat management** \> **Dashboard**.|[Security dashboard overview](https://docs.microsoft.com/en-us/office365/securitycompliance/security-dashboard)|
```
Comment (nenonix) [inline Exchange/ExchangeOnline/monitoring/use-mail-protection-reports.md]:
```suggestion
|**Threat management dashboard** (this is also referred to as the **Security dashboard** and the **Threat Intelligence dashboard**). <br/> Threat detections, malware trends, top targeted users, details about sent and received email messages, and more.|In the Security & Compliance Center, go to **Threat management** \> **Dashboard**.|[Security dashboard overview](https://docs.microsoft.com/office365/securitycompliance/security-dashboard)|
```

File: Exchange/ExchangeOnline/monitoring/use-mail-protection-reports.md [hunk 4]
```diff
@@ -28,18 +28,18 @@ If you were viewing mail protection reports in the Exchange admin center, they'v
...
+|**Advanced Threat Protection and email security reports** <br/> Email security and threat protection reports (including malware, spam, phishing, and spoofing reports).| In the Security & Compliance Center, go to **Reports** > **Dashboard**.|[View reports for Office 365 Advanced Threat Protection](https://docs.microsoft.com/en-us/office365/securitycompliance/view-reports-for-atp)  <br/>  [View email security reports in the Security & Compliance Center](https://docs.microsoft.com/en-us/office365/securitycompliance/view-email-security-reports)|
```
Comment (nenonix) [inline Exchange/ExchangeOnline/monitoring/use-mail-protection-reports.md]:
```suggestion
|**Advanced Threat Protection and email security reports** <br/> Email security and threat protection reports (including malware, spam, phishing, and spoofing reports).| In the Security & Compliance Center, go to **Reports** > **Dashboard**.|[View reports for Office 365 Advanced Threat Protection](https://docs.microsoft.com/en-us/office365/securitycompliance/view-reports-for-atp)  <br/>  [View email security reports in the Security & Compliance Center](https://docs.microsoft.com/office365/securitycompliance/view-email-security-reports)|
```

File: Exchange/ExchangeOnline/monitoring/use-mail-protection-reports.md (hunk 1 above)
Comment (nenonix) [inline Exchange/ExchangeOnline/monitoring/use-mail-protection-reports.md]:
```suggestion
|**Mail flow** <br/> Information about sent and received email messages, recent alerts, top senders and recipients, email forwarding reports, and more .| In the Security & Compliance Center, go to **Mail flow** > **Dashboard**.|[Mail flow insights in the Office 365 Security & Compliance Center](https://docs.microsoft.com/office365/securitycompliance/mail-flow-insights-v2)|
```

File: Exchange/ExchangeOnline/monitoring/use-mail-protection-reports.md [hunk 5]
```diff
@@ -28,18 +28,18 @@ If you were viewing mail protection reports in the Exchange admin center, they'v
...
+|**Mail flow** <br/> Information about sent and received email messages, recent alerts, top senders and recipients, email forwarding reports, and more .| In the Security & Compliance Center, go to **Mail flow** > **Dashboard**.|[Mail flow insights in the Office 365 Security & Compliance Center](https://docs.microsoft.com/en-us/office365/securitycompliance/mail-flow-insights-v2)|
 
 ## Related topics
 
-[Reports and insights in the Office 365 Security & Compliance Center](https://support.office.com/article/e3e95f68-36e9-4256-bcca-78fe7fe5ea5d)
+[Reports and insights in the Office 365 Security & Compliance Center](https://docs.microsoft.com/en-us/office365/securitycompliance/reports-and-insights-in-security-and-compliance)
```
Comment (nenonix) [inline Exchange/ExchangeOnline/monitoring/use-mail-protection-reports.md]:
```suggestion
[Reports and insights in the Office 365 Security & Compliance Center](https://docs.microsoft.com/office365/securitycompliance/reports-and-insights-in-security-and-compliance)
```

File: Exchange/ExchangeOnline/monitoring/use-mail-protection-reports.md (hunk 2 above)
Comment (andypunt) [reply Exchange/ExchangeOnline/monitoring/use-mail-protection-reports.md]: looks good to me.

File: Exchange/ExchangeOnline/monitoring/use-mail-protection-reports.md (hunk 3 above)
Comment (andypunt) [reply Exchange/ExchangeOnline/monitoring/use-mail-protection-reports.md]: ack

File: Exchange/ExchangeOnline/monitoring/use-mail-protection-reports.md (hunk 4 above)
Comment (andypunt) [reply Exchange/ExchangeOnline/monitoring/use-mail-protection-reports.md]: ack

File: Exchange/ExchangeOnline/monitoring/use-mail-protection-reports.md (hunk 5 above)
Comment (andypunt) [reply Exchange/ExchangeOnline/monitoring/use-mail-protection-reports.md]: ack

**Pull request merged**